grouper/fe/handlers/permissions_revoke.py

```diff
@@ -18,13 +18,12 @@ def check_access(session, mapping, user):
 
         grantable = user_grantable_permissions(session, user)
 
-        can_grant = False
         for perm in grantable:
             if perm[0].name == mapping.permission.name:
                 if matches_glob(perm[1], mapping.argument):
-                    can_grant = True
+                    return True
```

Comment:
you could *also* implement this as:
```
return any([matches_glob(perm[1], mapping.argument) for perm in grantable if perm[0].name == mapping.permission.name])
```

Not sayin' ya should, mind you. :trollface: